src/containers/tutor-home-page/general-info-step/SelectGroup.tsx

```diff
@@ -1,30 +1,78 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { FormControl, InputLabel, Select, MenuItem } from '@mui/material'
 import { useTranslation } from 'react-i18next'
 import { useSelectGroupStyles } from './SelectGroup.styles'
+import { LocationService } from '~/services/location-service'
+interface Country {
+  name: string
+}
 
+interface City {
+  name: string
+}
 const SelectGroup: React.FC = () => {
   const { t } = useTranslation()
   const classes = useSelectGroupStyles()
+  const [countries, setCountries] = useState([])
```

Comment:
```suggestion
  const [countries, setCountries] = useState<string[]>([])
```

LocationService.getCountries() service - returns an array of strings.

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/56509285/af4d925c-5f9a-4023-9d9b-a11fc3c5f554)